README.md

```diff
@@ -284,6 +286,22 @@ void handleBody(AsyncWebServerRequest *request, uint8_t *data, size_t len, size_
   }
 }
 ```
+If needed, the `_tempObject` field on the request can be used to store a pointer to temporary data (e.g. from the body) associated with the request. If assigned, the pointer will automatically be freed along with the request.
+
+### JSON body handling with ArduinoJson
+Endpoints which consume JSON can use a special handler to get ready to use JSON data in the request callback:
+```cpp
+#include "AsyncJson.h"
+#include "ArduinoJson.h"
+
+AsyncCallbackJsonWebHandler* handler = new AsyncCallbackJsonWebHandler();
+handler->setUri("/rest/endpoint");
```

Comment:
I'm thinking that the URI should be a part of the AsyncCallbackJsonWebHandler constructor (almost everything else in the lib works that way)